src/main/java/hudson/plugins/logparser/LogParserDisplayConsts.java

```diff
@@ -11,9 +13,13 @@
 
     public LogParserDisplayConsts() {
         // Color of each status
-        colorTable.put(LogParserConsts.ERROR, "red");
-        colorTable.put(LogParserConsts.WARNING, "orange");
-        colorTable.put(LogParserConsts.INFO, "blue");
+        LogParserPublisher.DescriptorImpl descriptor = (LogParserPublisher.DescriptorImpl) Jenkins.getInstance().getDescriptor(LogParserPublisher.class);
+        String colorError = descriptor.getColorError();
+        String colorWarning = descriptor.getColorWarning();
+        String colorInfo = descriptor.getColorInfo();
+        colorTable.put(LogParserConsts.ERROR, colorError == null || colorError.isEmpty() ? "red" : colorError);
```

Comment:
Thanks for the ultra quick feedback, agreed on all your comments :) will make the changes.